docs/source/index.rst

```diff
@@ -25,7 +25,6 @@ Support for Windows is possible in the near future.
 
    cuml_intro.rst
    cuml_blogs.rst
-   estimator_intro.ipynb
```

Comment:
This is only removed for debugging docs quicker temporarily, will be added back before merging